pasqal_cloud/batch.py

```diff
@@ -63,36 +68,38 @@ class Batch(BaseModel):
     user_id: str
     priority: int
     status: str
-    webhook: Optional[str]
-    _client: Client
+    _client: Client = PrivateAttr(default=None)
     sequence_builder: str
-    start_datetime: Optional[str]
-    end_datetime: Optional[str]
-    device_status: Optional[str]
     ordered_jobs: List[Job] = []
     jobs_count: int = 0
     jobs_count_per_status: Dict[str, int] = {}
+    webhook: Optional[str] = None
+    start_datetime: Optional[str] = None
+    end_datetime: Optional[str] = None
+    device_status: Optional[str] = None
     parent_id: Optional[str] = None
     configuration: Union[BaseConfig, Dict[str, Any], None] = None
 
-    class Config:
-        extra = Extra.allow
-        arbitrary_types_allowed = True
+    model_config = ConfigDict(extra="allow", arbitrary_types_allowed=True)
 
-    @root_validator(pre=True)
-    def _build_ordered_jobs(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """This root validator will modify the 'jobs' attribute which is a list
+    def __init__(self, **data: Any) -> None:
+        """
+        Makes sure the '_client' is set when instantiating a Batch
```

Comment:
This docstring is not 100% accurate, pydantic does support private attributes but one needs to use this workaround to make it work. I would link to the pydantic documentation where they do the same thing as here to instantiate their private fields.